docs/source/API/cli.rst

```diff
@@ -181,8 +181,9 @@ Examples:
 
 .. option:: --n_queries N_QUERIES
 
-    The number of queries. By default, the program stops after all documents are reviewed
-    or is interrupted by the user.
+    The number of queries. Alternatively, entering :code:`min` will stop the simulation when all relevant
+    documents have been found. By default, the program stops after all documents are reviewed
```

Comment:
Can you change ' documents'  into 'records'? See our vocabulary: https://asreview.readthedocs.io/en/latest/intro/vocabulary.html